lichess-bot.py

```diff
@@ -732,6 +855,12 @@ def print_pgn_game_record(li: lichess.Lichess, config: Configuration, game: mode
 
 
 def fill_missing_pgn_headers(game_record: chess.pgn.Game, game: model.Game) -> None:
+    """
+    Fill the headers missing by the lichess.org PGN with local headers from `game`.
```

Comment:
Fill in any missing headers in the PGN record provided by lichess.org with information from `game`.